jscript.js

```diff
@@ -37,4 +36,226 @@ function closemenu() {
   main.className = 'head flex section-one background-white';
 }
 clbtn.addEventListener('click', closemenu);
-uli.addEventListener('click', closemenu);
\ No newline at end of file
+uli.addEventListener('click', closemenu);
+
+// dynamic content
+const projects = [
+  {
+    projectName: `
+    <h1>Tonic</h1>
+    <p class = "close">x</p>`,
+    listoficon: `    
+    <p>CANOPY</p>
+    <img src = "./img/counter.png" width = 10 height = 10>
+    <p>Back End Dev</p>
+    <img src = "./img/counter.png" width = 10 height = 10>
+    <p>2015</p>`,
+    projectDescription: `
+    
+    <div class = "description-p">
+      <p> Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+      Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown 
+      printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the 
+      printing and typesetting industry. </p><br>
+
+      <p>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer 
+      took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing 
+      and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the 
+      releawn printer took a galley of type and scrambled it 1960s. </p>
+    </div>
```

Comment:
- [ ] You have done very nice work to have a good design and a functioning window popup. However, we need to implement the use of [array of objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#using_map_to_reformat_objects_in_an_array). Therefore the objects should have the key and value which is not HTML tags. Kindly consider following this example to refactor your array of objects and use them properly.

`
const projects = [{
    name: "tomic",
    description: "This is tonic description",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "css"],
    link: "live link",
}]
`